ValidationRelay/ContentView.swift

```diff
@@ -10,49 +10,44 @@ import SwiftUI
 struct ContentView: View {
     @AppStorage("autoConnect") private var wantRelayConnected = false
     @AppStorage("keepAwake") private var keepAwake = true
-    
     @AppStorage("selectedRelay") private var selectedRelay = "Beeper"
     @AppStorage("customRelayURL") private var customRelayURL = ""
-    
-    @ObservedObject var relayConnectionManager: RelayConnectionManager
-    
-    init(relayConnectionManager: RelayConnectionManager) {
-        self.relayConnectionManager = relayConnectionManager
-        if wantRelayConnected {
-            relayConnectionManager.connect(getCurrentRelayURL())
-        }
-        if keepAwake {
-            UIApplication.shared.isIdleTimerDisabled = true
-        }
+
+    @StateObject private var relayConnectionManager = RelayConnectionManager()
```

Comment:
```swift
ValidationRelayApp.swift:14:49: error: argument passed to call that takes no arguments
            ContentView(relayConnectionManager: RelayConnectionManager())
                       ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~
```
You forgot to remove the argument in https://github.com/cameronaaron/ValidationRelay/blob/f549171f637871837321c257d4dd2d35c8f3b7be/ValidationRelay/ValidationRelayApp.swift#L14 where this is constructed